servers.json

```diff
@@ -876,7 +876,9 @@
         "id": "minelord",
         "name": "MineLord Network",
         "addresses": [
-            "minelord.org"
+            "minelord.org",
+            "play.minelord.org",
```

Comment:
Lunar Client services automatically detect sub-domains, this commit is useless.

Refer to the [ip-addresses](https://github.com/LunarClient/ServerMappings#ip-addresses) section of this repository's read-me for more information.